tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb

```diff
@@ -0,0 +1,476 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4c261298-309f-41e8-9338-a5e205f09b05",
+   "metadata": {},
+   "source": [
+    "# Post Training Quantization a Pytorch Object Detection Model - A Quick-Start Guide\n",
+    "\n",
+    "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb)\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "This tutorial shows how to quantize a pre-trained object detection model from the torchvision package using the Model-Compression-Toolkit (MCT). We will do so by giving an example of MCT's post-training quantization. As we will see, post-training quantization is a low complexity yet effective quantization method. In this example, we will quantize the model and evaluate the accuracy before and after quantization.\n",
+    "\n",
+    "As the pretrained object detection model contains a preprocessing and postprocessing layers that their quantization with MCT is out of this notebook's scope. We'll separate these layers from the model-to-quantize. These layers will be included in the evaluation code.\n",
```

Comment:
you need comma instead of period
As the... out this notebook's scope, we'll separate ...